### DISCUSSION TITLE
LLaVA vs mplugOwl image tagging

### DISCUSSION THREAD

**Discussion by segalinc** [PixArt-alpha/PixArt-alpha#32] [General]
Hi and thanks for the amazing contribution
I am curious to hear if you also tried or compared MplugOwl visual LLM to refine the image captions other than LLaVA and which one you saw is better, pros and cons 

thank you

**Comment by yujincheng08** [comment]
no.

**Comment by lawrence-cj** [comment]
We are not planning to do such comparison since there are more and more powerful VLMs, such as LLaVA 1.5 or Mini-GPT v2. Maybe directly update the LLaVA-captions to more powerful captioners is a better choice.
